src/Products/Viewer/Controllers/ViewerApiController.cs

```diff
@@ -49,6 +53,9 @@ public ViewerApiController()
             {
                 fontsDirectory.Add(globalConfiguration.Viewer.GetFontsDirectory());
             }
+
+            //this.FileWrapper = new LocalFileWrapper(globalConfiguration);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Viewer-for-.NET-MVC/pullRequest?prid=6117782)